OlinkAnalyze/R/Read_NPX_data.R

```diff
@@ -306,6 +315,27 @@ read_NPX_target <- function(filename) {
                     dplyr::matches("*Inc Ctrl*"),
                     dplyr::matches("*Det Ctrl*"))
     is_npx_data <- ifelse(any(names(out) %in% "NPX"), TRUE, FALSE)
+  } else  if (long_form == TRUE && isTarget == FALSE){
+    out <- readxl::read_excel(path = filename,
+                              col_names = T) %>%
+      dplyr::filter(!is.na(SampleID)) %>%
+      dplyr::as_tibble() %>%
+      dplyr::select(SampleID, Index, OlinkID,
+                    UniProt, Assay, MissingFreq,
+                    Panel,Panel_Version,PlateID,
+                    QC_Warning,dplyr::matches("Plate_LQL"),
+                    dplyr::matches("LOD"),
+                    dplyr::matches("Plat_LOD"),
+                    dplyr::matches("LLOQ"),
+                    dplyr::matches("ULOQ"),
+                    dplyr::matches("NPX"),
+                    dplyr::matches("Quantified_value"),
+                    dplyr::matches("Unit"),
+                    dplyr::matches("Assay_Warning"),
+                    dplyr::matches("Normalization"),
+                    dplyr::matches("*Inc Ctrl*"),
+                    dplyr::matches("*Det Ctrl*"))

```

Comment:
Would `tidyselect::any_of()` work here?

Reply:
It wasnt working for me but I dont know why.